src/carbon_calculator/carbonCalculator.py

```diff
@@ -26,6 +28,47 @@
 from .landscaping import EvalReduceLawnSize, EvalReduceLawnCare, EvalRakeOrElecBlower, EvalElectricMower
 #from .calcUsers import ExportCalcUsers, CalcUserUpdate
 
+CALCULATOR_VERSION = "4.0.1"
+QUESTIONS_DATA = 'carbon_calculator/content/Questions.csv'
+ACTIONS_DATA = 'carbon_calculator/content/Actions.csv'
+DEFAULTS_DATA = 'carbon_calculator/content/Defaults.csv'
+
+def fileDateTime(path):
+    # file modification
+    timestamp = os.path.getmtime(path)
+    utc=pytz.UTC
+
+    # convert timestamp into DateTime object
+    datestamp = datetime.fromtimestamp(timestamp).replace(tzinfo=utc)
+    return datestamp
+
+def versionCheck():
+    version = Version.objects.all()
+    if not version:
+        version = Version.objects.create(version=CALCULATOR_VERSION, note="Initial Version")
+        print(version.note)
+        return False    # reload data
+    else:
+        version = version.first()
```

Comment:
If we have multiple versions. This code wouldn't return the latest

Reply:
There shouldn't be multiple versions, but in case there are I changed it to .latest('id')